src/main/java/app/commands/DeleteContactsCommand.java

```diff
@@ -1,21 +1,26 @@
 package app.commands;
 
+import app.exception.AppException;
 import app.sql.dao.mysql.ContactDaoImpl;
 
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import java.util.HashSet;
+import java.util.Arrays;
 import java.util.Set;
+import java.util.stream.Collectors;
 
 public class DeleteContactsCommand implements ActionCommand {
     @Override
     public String execute(HttpServletRequest request, HttpServletResponse response) {
-        String[] contacts = request.getParameterMap().get("id");
-        Set<Integer> ids = new HashSet<>();
-        for (String parameter : contacts) {
-            ids.add(Integer.valueOf(parameter));
+        try {
+        Set<Integer> contactIds = Arrays.stream(request.getParameterMap().get("id"))
+                .map(Integer::valueOf).collect(Collectors.toSet());
```

Comment:
It looks strange that you use `Integer::valueOf` and haven't used the same syntax for `Collectors.toSet()`